target/linux/ath79/image/generic.mk

```diff
@@ -1180,6 +1180,22 @@ define Device/meraki_mr16
 endef
 TARGET_DEVICES += meraki_mr16
 
+define Device/meraki_mr12
```

Comment:
mr12 should be before mr16 ...

Reply:
Done.

Reply:
[mr12 is before mr16](https://github.com/openwrt/openwrt/pull/3634/commits/e783ef483365f08712c29411b2a73891e5cc2ae8#diff-a708a78acb0be33ea1ccf60a3e280671314041a1e2e497bd90cad9386f4518bbR1336). Marking as resolved.